bin/install_travis.sh

```diff
@@ -1,37 +1,82 @@
 #!/usr/bin/env bash
 
-sudo apt-get update
-sudo apt-get install cmake libgmp-dev
+if [[ "${CC}" == "" ]]; then
+    export CC=gcc
+    export CXX=g++
+fi
+if [[ "${TRAVIS_OS_NAME}" == "osx" ]] && [[ "${CC}" == "gcc" ]]; then
+    export CC=gcc-4.8
+    export CXX=g++-4.8
```

Comment:
`gcc` is just a wrapper around `clang` whereas `gcc-4.8` is a true `gcc`.

https://github.com/travis-ci/travis-ci/issues/2423